.github/workflows/build.yml

```diff
@@ -56,9 +65,10 @@ jobs:
 
     - name: Build and test
       run: |
-        xcodebuild build test -project ${project} -scheme "${scheme}" ${params} -destination ${destination} | xcpretty --utf --color && exit ${PIPESTATUS[0]}
+        xcodebuild build test -project ${project} -scheme "${scheme}" ${params} -destination "${destination}" | xcpretty --utf --color && exit ${PIPESTATUS[0]}
       env:
         project: 'Adyen.xcodeproj'
         params: "-skipPackagePluginValidation"
-        scheme: ${{ matrix.scheme }}
+        scheme: 'AdyenUIKitTests'
         destination: "OS=${{ matrix.version }},name=${{ matrix.displayname }}"
```

Comment:
For `device: 'iPhone 13'` and  `displayname: 'iPhone-13` we can use 

```yaml
    matrix:
        include:
            - iphone: 13
            - iphone: 12
            
           env:
                displayname: iPhone-${{ matrix.displayname }}
                device: 'iPhone ${{ matrix.displayname }}'
            
          env:
               destination: "OS=${{ matrix.version }},name= iPhone -${{ matrix.iphone }}"
```

Reply:
Interesting one - would create a PR that does some more fixes to the `build.yml` and test it in isolation 👌